src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -603,16 +630,16 @@ private void handleReset(
         {
             try
             {
-                correlations.remove(applicationCorrelationId);
+                correlations.remove(networkCorrelationId);
                 doCloseInbound(tlsEngine);
             }
             catch (SSLException ex)
             {
-                LangUtil.rethrowUnchecked(ex);
+                // ignore and cleanup
             }
             finally
             {
-                networkThrottle.accept(reset.typeId(), reset.buffer(), reset.offset(), reset.sizeof());
+                doReset(applicationThrottle, applicationId);
```

Comment:
The `reaktor` will implicitly `RESET` if we successfully remove the correlation, so this seems like it could trigger a duplicate `RESET` for that case.

Perhaps change the `try` block as follows:
```
if (correlations.remove(networkCorrelationId) == null)
{
    doReset(applicationThrottle, applicationId);
}
doCloseInbound(tlsEngine);
```
...and remove the `finally` block.